resources/js/visual-editor/blocks/button/button.css

```
@@ -0,0 +1,136 @@
+ *
+ * Vendored from `@wordpress/block-library/src/button/{style,editor}.scss`
+ * (v9.43.0). SCSS variables inlined as 0.5em; `data-type="core/button"`
+ * selectors rewritten to `data-type="artisanpack/button"`.
+ */
+
+/* style.scss */
+.wp-block-button__link {
+    cursor: pointer;
+    display: inline-block;
+    text-align: center;
+    word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)